src/cdk-experimental/menu/menu-bar.spec.ts

```diff
@@ -735,6 +737,108 @@ describe('MenuBar', () => {
       );
     });
   });
+
+  describe('background click closeout', () => {
+    let fixture: ComponentFixture<MenuBarWithMenus>;
+
+    let menus: CdkMenu[];
+    let triggers: CdkMenuItemTrigger[];
+
+    /** open the attached menu. */
+    const openMenu = () => {
```

Comment:
Nit: our internal style guide recommends using 'function openMenu() {' instead of assigning lambda functions to variables.